src/util/NetworkStatus.cpp

```diff
@@ -45,20 +45,20 @@ void NetworkStatus::fromJsonObject(const Json::Value& object)
 
     time_t raw_time;
     time(&raw_time);
-    m_savedDate = QString(ctime(&raw_time));
-    m_savedDate = m_savedDate.trimmed();
+    //TODO: Trim savedDate
```

Comment:
Please add trim implementation in string util, and use it here. Otherwise the logs in the error page will be harder to read.

Reply:
Done.